### ISSUE TITLE
Can't declare superclass of cpp:class

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#678]
The python domain allows superclasses to be declared with the same syntax as python:

```
.. py:class:: Sphinx(DocGenerator)
```

cpp:class does not allow the superclass to be declared:

```
.. cpp:class:: Sphinx : public DocGenerator
```

C++ superclasses come after a single colon (which should be unambiguous in this context). One or may be specified, separated by commas. Each may optionally be preceded by a visibility modifier; the default is private, so it's pretty common for `public` to be present.

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/678
- Originally reported by: Anonymous
- Originally created at: 2011-04-17T07:10:10.988

**Comment by shimizukawa** [comment]
_From Robert Lehmann on 2011-10-08 12:12:29+00:00_

Fixes #678: Support superclass declarations in C++ domain.

→ <<cset 9904596da8c84007f579d9ca9e119468d02cf5fd>>

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2014-10-18 07:32:18+00:00_

Removing version: 1.0.7 (automated comment)
